### ISSUE TITLE
Bazel workspace.bzl requires a dependency on rules_closure

### ISSUE THREAD

**Issue by alexeagle** [tensorflow/tensorflow#32556]
Per https://stackoverflow.com/questions/52482972/running-load-within-skylark-macro
users of Tensorflow are adding rules_closure to their `WORKSPACE` file even though they don't use it.

rules_closure is minimally maintained (my team has just acquired it). We don't want new dependencies on it and have discovered that most dependencies are due to this issue.

I suspect the Tensorflow users don't actually want to build a Closure Compiler-minified UI, so any `load()` statement from rules_closure doesn't belong in the tensorflow bazel rules distribution.

Note, #15997 suggests customizing the dependencies, but in this issue I propose that users should never observe this dependency at all.

**Comment by angerson** [comment]
Thanks for mentioning this. I don't think I quite follow you though, what exactly can we do to avoid this without making our own files more confusing?

**Comment by alexeagle** [comment]
The typical way we handle this in rules_nodejs is to "fence" regions of BUILD files that are only for our internal use, then strip these from the published distribution artifact.
How exactly to do this depends on how you build the Tensorflow distribution. I'm happy to help fix this if you can give some pointers to where the distribution is built.

**Comment by kumariko** [comment]
@alexeagle Could you please let us know if you still need help on this ? if it is resolved then please feel free to move this issue to close status ? Thanks!

**Comment by alexeagle** [comment]
yes I think this is still a problem, though I don't know where the workspace.bzl file is now, nor what instructions you give Bazel users. @gkdn has been maintaining rules_closure lately and might be closer to understanding the impact.

**Comment by gadagashwini** [comment]
@alexeagle, 
You can find the workspace.bzl file [here](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/workspace0.bzl).
Thanks!

**Comment by alexeagle** [comment]
Activity for the bot!

**Comment by tilakrayal** [comment]
Hi,

Thank you for opening this issue. Since this issue has been open for a long time, the code/debug information for this issue may not be relevant with the current state of the code base.

The Tensorflow team is constantly improving the framework by fixing bugs and adding new features. We suggest you try the latest TensorFlow [version](https://pypi.org/project/tensorflow/) with the latest compatible hardware configuration which could potentially resolve the issue. If you are still facing the issue, please create a new GitHub issue with your latest findings, with all the debugging information which could help us investigate.

Please follow the [release notes](https://github.com/tensorflow/tensorflow/blob/master/RELEASE.md) to stay up to date with the latest developments which are happening in the Tensorflow space.

**Comment by Venkat6871** [comment]
Hi @alexeagle ,
Thank you so much for filing this build/install request! We truly appreciate your support. At the moment, this request is not prioritized and mark it for community contributions. If you would like to contribute on a fix, please feel free to fork the repository and submit a pull request in accordance with the TensorFlow [CONTRIBUTION GUIDELINES](https://www.tensorflow.org/community/contribute/code) and link this issue in your PR to ensure the maintainers can easily review and collaborate.To avoid duplicate work, please leave a comment here if you decide to take it on!
Thank you again for your interest and we look forward to seeing your contributions!
